packages/nimble-components/src/spinner/styles.ts

```diff
@@ -0,0 +1,216 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    Black15,
+    Black91,
+    White
+} from '@ni/nimble-tokens/dist/styledictionary/js/tokens';
+import { Theme } from '../theme-provider/types';
+import { hexToRgbaCssColor } from '../utilities/style/colors';
+import { themeBehavior } from '../utilities/style/theme';
+
+export const styles = css`
+    ${display('inline-block')}
+
+    :host {
+        width: 32px;
+        height: 32px;
+    }
+
+    :host([size='small']) {
+        width: 16px;
+        height: 16px;
+    }
+
+    :host([size='medium-large']) {
+        width: 48px;
+        height: 48px;
+    }
+
+    :host([size='large']) {
+        width: 64px;
+        height: 64px;
+    }
+
+    :host([size='x-large']) {
+        width: 96px;
+        height: 96px;
+    }
+
+    :host([size='xx-large']) {
+        width: 128px;
+        height: 128px;
+    }
+
+    div.container {
+        width: 87.5%;
+        height: 87.5%;
+        top: 6.25%;
+        left: 6.25%;
```

Comment:
I'm assuming the spinner is always centered in its box right? Can we use flex layout on the host so we can just center the .container? Then we don't need to worry about the top, left offset, just the relative widths

Reply:
Done